### PR TITLE
Dashboard UI changes

### PR DESCRIPTION
**Pull request by satyamkumar-cn**

### CODE REVIEW

**Comment by amit-chaurasia** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

**Comment by amit-chaurasia** [comment]
:white_check_mark: No issues identified in this pull request. 

**Good work!**
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR increased your FTAR from 0.00 % to 100.00 %, good job, and for more quality improvements you can read about yellow findings.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>100.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>0</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

**Comment by amit-chaurasia** [comment]
You are not receiving code review NPS requests in your digest because you don't have your work email address specified in your [GitHub profile](https://github.com/satyamkumar-cn).
<details>
<summary>More information</summary>
<table><tr><td>
To avoid this please follow the following steps:

1. Add your corporate email address on GitHub via [this link](https://github.com/settings/emails (a verification of an email is required))
2. Uncheck the "Keep my email address private" option
3. Switch to [profile settings](https://github.com/settings/profile) and select your corporate email address in the "Public email" drop-box
4. Click "Update profile"
</td></tr></table>
</details>
<br>

Hi, I'm Amit Chaurasia from [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members) and I have just reviewed your PR.
<b>How likely are you to recommend the Code Review service to another developer?</b>

Not at all likely \[ [1](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=1) \] \[ [2](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=2) \] \[ [3](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=3) \] \[ [4](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=4) \] \[ [5](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=5) \] \[ [6](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=6) \] \[ [7](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=7) \] \[ [8](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=8) \] \[ [9](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=9) \] \[ [10](https://docs.google.com/a/aurea.com/forms/d/e/1FAIpQLSdEMTCUcOnA4z86LUOWodIg0M6XQpQT83QtLRg_xgjeHOPPoA/viewform?entry.113507233=https://github.com/trilogy-group/gitpod/pull/8&entry.1163746017=10) \] Very likely

> TIP: Press `Ctrl` (or `⌘` in a Mac) to open the survey in a new tab.

File: components/dashboard/src/index.tsx
Comment (pavan-tri) [inline components/dashboard/src/index.tsx:18]: Any reason for wrapping this with Router ?

File: components/dashboard/src/components/Menu.tsx
Comment (pavan-tri) [inline components/dashboard/src/components/Menu.tsx:54]: Try to keep the structure very similar to whats available; If its left and right and we want things in left, lets not rename to list.

File: components/dashboard/src/App.tsx
Comment (pavan-tri) [inline components/dashboard/src/App.tsx:156]: Dont change to list, keep it left

File: components/dashboard/src/App.tsx
Comment (pavan-tri) [inline components/dashboard/src/App.tsx:177]: Take the values from Branding config in Yaml

File: components/dashboard/src/App.tsx
Comment (pavan-tri) [inline components/dashboard/src/App.tsx:146]: Why did we remove BrowserRouter and add added Router  on top of App ?

File: components/dashboard/public/manifest.json
Comment (pavan-tri) [inline components/dashboard/public/manifest.json:2]: Can't change this hardcoded. Have to come from banding YAML or method to configure via build script.

File: components/dashboard/public/index.html
Comment (pavan-tri) [inline components/dashboard/public/index.html:11]: Can't change hardcoded, has to come via a build config or PUBLIC_ICON_URL etc

**Review by pavan-tri — CHANGES_REQUESTED** [review]
#1 Don't remove any existing files. revert them.
#2 Make it config driven.
#3 Try to keep the structure similar don't make optimisation or refactor changes. As this port is meant to be used by others as well.

File: components/dashboard/src/App.tsx
Comment (satyamkumar-cn) [reply components/dashboard/src/App.tsx:146]: Actually Router is just an alias for BrowserRouter. Anyways I will change it back.

File: components/dashboard/src/components/Menu.tsx
Comment (satyamkumar-cn) [reply components/dashboard/src/components/Menu.tsx:54]: Okay.

File: components/dashboard/script.js
Comment (pavan-tri) [inline components/dashboard/script.js:5]: This can't be hardcoded. this should come from ENV or config

**Review by pavan-tri — CHANGES_REQUESTED** [review]
Don't format the spacing its difficult to review the files

File: components/dashboard/script.js
Comment (satyamkumar-cn) [reply components/dashboard/script.js:5]: See The way I am doing it is you just need to change your desired values here in this file. So these are default values . This script writes a .env file. And then those are used elsewhere in the code. So in this this script.js is one point config spot.

**Comment by vdefeo-xo** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

File: components/dashboard/script.js
Comment (vdefeo-xo) [inline components/dashboard/script.js:16]: <details>
<summary>:warning: <strong>This code contains single line control structures without curly braces, `{}`.</strong> (click for details)</summary>

Consider surrounding these statements with `{ }`.  
  
Braces are used around all statements, even singletons, when they are part of a control structure, such as an `if`-`else` or `for` statement.  
  
This makes it easier to add statements without accidentally introducing bugs due to forgetting to add braces. Readability is also significantly improved.

**[More details](https://confluence.devfactory.com/x/fHvlFQ)**

## The improvement suggestion in this case
```suggestion
for (let prop in vars) {
  data += prop + "=" + vars[prop] + "\n";
}
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="577f3a94-c7d6-498b-bc93-fda1fe27cfce" data-finding-id="367360892" href="none" codereview-uuid-reserved-tag data-business-id="507be860-41ae-46a0-83e9-7f8e359e3039" data-analyzer="js" />)

File: components/dashboard/src/App.tsx
Comment (vdefeo-xo) [inline components/dashboard/src/App.tsx:9]: <details>
<summary>:warning: <strong>This code is using `"` double-quotes in `require` or `import` statement.</strong> (click for details)</summary>

A possible improvement would be to follow best conventions of JS and use `'` single-quote to declare path in `require` statements.  
  
This change would make the code clearer and consistent.

**[More details](https://confluence.devfactory.com/x/DXFnFQ)**

## The improvement suggestion in this case
```tsx  
import { BrowserRouter } from 'react-router-dom';
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="fc921aaf-17d0-46ee-8a8a-dafd9fed9a3f" data-finding-id="359100685" href="none" codereview-uuid-reserved-tag data-business-id="507be860-41ae-46a0-83e9-7f8e359e3039" data-analyzer="js" />)

File: components/dashboard/src/App.tsx
Comment (vdefeo-xo) [inline components/dashboard/src/App.tsx:49]: <details>
<summary>:warning: <strong>This code contains a statement without a semicolon.</strong> (click for details)</summary>

A possible improvement would be to keep the code consistency and insert the semicolon at the end of the line.  
  
This change would ensure the code follows the current code standard, making it easier for new developers to start without the need to learn a different convention and, thus, improving maintainability and keeping the code consistency.

**[More details](https://confluence.devfactory.com/x/lAsaF)**

## The improvement suggestion in this case
Consider adding the semicolon at the end of the line.  
  
```suggestion  
                 const usr = await getGitpodService().server.getLoggedInUser();
```
</details>

[//]: # (<a id="623b5be1-8d15-408b-b60f-a58e03b87463" data-finding-id="337251220" href="none" codereview-uuid-reserved-tag data-business-id="507be860-41ae-46a0-83e9-7f8e359e3039" data-analyzer="js" />)

File: components/dashboard/src/App.tsx
Comment (vdefeo-xo) [inline components/dashboard/src/App.tsx:52]: <details>
<summary>:warning: <strong>This code seems to be logging information in production - console.log()</strong> (click for details)</summary>

Generally, using `console.log` instead of `alert()` is a good idea for debugging purposes but this can bloat your JavaScript files and also leak out sensitive information of your app as anybody can read application logs by opening dev-tools.   
An important improvement would be to remove this logging statement from the code.  
  
This change would make your app more secure, less prone to information leakage and cleaner.

**[More details](https://confluence.devfactory.com/x/AwIGF)**

## The improvement suggestion in this case
Consider removing `console.log` statements.
</details>

[//]: # (<a id="c76dcba2-2205-4df6-80dd-9599398b5305" data-finding-id="335938051" href="none" codereview-uuid-reserved-tag data-business-id="507be860-41ae-46a0-83e9-7f8e359e3039" data-analyzer="js" />)

File: components/dashboard/src/App.tsx
Comment (vdefeo-xo) [inline components/dashboard/src/App.tsx:123]: <details>
<summary>:warning: <strong>Commented-out Code is Committed.</strong> (click for details)</summary>

This pull request contains commented code which is not recommended when a version control repository (e.g., GitHub) is used. It is recommended to delete all commented instructions before committing to VCS.

**[More details](https://confluence.devfactory.com/x/yAV7HQ)**

## The improvement suggestion in this case
Consider avoiding committing commented code.
</details>

[//]: # (<a id="c483fce2-a802-4337-b694-0393c5c08620" data-finding-id="494601672" href="none" codereview-uuid-reserved-tag data-business-id="507be860-41ae-46a0-83e9-7f8e359e3039" data-analyzer="js" />)

**Review by vdefeo-xo — COMMENTED** [review]
Hi, I am the JavaScript expert who performed the review of this Pull Request. We have well-defined [coding and review standards](https://confluence.devfactory.com/display/CodeReview/Findings), and you will find below suggestions to improve the code quality, based on these guidelines.
My additional comments are in the Notes section. I hope you find them useful.

<table>
  <tr>
    <td>:no_entry_sign:<b>Severes:</b>  0</td>
    <td>:exclamation:<b>Reds:</b>  0</td>
    <td>:warning:<b>Yellows:</b>  7</td>
    <td>:white_check_mark:<b>Greens:</b>  0</td>
  </tr>
</table>

<details><summary>More information</summary>
  <table>
    <tr>
      <td>:no_entry_sign:</td>
      <td><b>Severe Findings</b> point out serious violations which can affect stability, performance, maintainability and therefore should be fixed.
        <ul>
          <li>Please note that the merge might have been blocked until all Severe Findings are fixed.</li>
          <li>While it is highly recommended to implement all findings only Severe Findings might have been blocking the merge</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>:exclamation:</td>
      <td><b>Red Findings</b> represent potential bugs and it is highly recommended to implement them.<br/>
      Not required for the approval of the PR
      </td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td><b>Yellows</b> are suggestions to improve the code, warnings and tips<br/>
      Not required for the approval of the PR, and should not be used for FTAR calculation.
      </td>
    </tr>
    <tr>
      <td>:white_check_mark:</td>
      <td><b>Green Findings</b> highlight good ideas and solutions</td>
    </tr>
    <tr>
      <td>:repeat:</td>
      <td><b>Repeated</b> instances of the same findings are counted just once and flagged with the repeat icon</td>
    </tr>
  </table>

  Feel free to reply to this comment, (make sure you `@tag` us) so we can start a conversation. The goal here is to learn and grow together and make our products better.

  > NOTE: This review **does not** replace your team's review._
</details>
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR increased your FTAR from 0.00 % to 100.00 %, good job, and for more quality improvements you can read about yellow findings.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>100.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>0</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

**Comment by vdefeo-xo** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

**Comment by vdefeo-xo** [comment]
:white_check_mark: No new issues have been introduced since the previous review.

**Good work!**
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR increased your FTAR from 0.00 % to 100.00 %, good job, and for more quality improvements you can read about yellow findings.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>100.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>0</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

**Comment by vdefeo-xo** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

File: components/dashboard/script.js
Comment (vdefeo-xo) [inline components/dashboard/script.js:15]: <details>
<summary>:warning: <strong>This code uses `let` or `var` to declare one or more variables that are never reassigned.</strong> (click for details)</summary>

A possible improvement would be to declare such a variable with `const` instead.   
  
This change would ensure that it would not be reassigned unintentionally, which could lead to difficulty to comprehend bugs.

**[More details](https://confluence.devfactory.com/x/UA_kEw)**

## The improvement suggestion in this case
```js  
for (const prop in vars) {
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence.
</details>

[//]: # (<a id="8c04c96f-a100-47a4-9efa-25a078d652fc" data-finding-id="329518928" href="none" codereview-uuid-reserved-tag data-business-id="a365ef85-720e-4b91-8367-2fe7ee0de9a5" data-analyzer="js" />)

**Review by vdefeo-xo — COMMENTED** [review]
Hi, I am the JavaScript expert who performed the review of this Pull Request. We have well-defined [coding and review standards](https://confluence.devfactory.com/display/CodeReview/Findings), and you will find below suggestions to improve the code quality, based on these guidelines.
My additional comments are in the Notes section. I hope you find them useful.

<table>
  <tr>
    <td>:no_entry_sign:<b>Severes:</b>  0</td>
    <td>:exclamation:<b>Reds:</b>  0</td>
    <td>:warning:<b>Yellows:</b>  1</td>
    <td>:white_check_mark:<b>Greens:</b>  0</td>
  </tr>
</table>

<details><summary>More information</summary>
  <table>
    <tr>
      <td>:no_entry_sign:</td>
      <td><b>Severe Findings</b> point out serious violations which can affect stability, performance, maintainability and therefore should be fixed.
        <ul>
          <li>Please note that the merge might have been blocked until all Severe Findings are fixed.</li>
          <li>While it is highly recommended to implement all findings only Severe Findings might have been blocking the merge</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>:exclamation:</td>
      <td><b>Red Findings</b> represent potential bugs and it is highly recommended to implement them.<br/>
      Not required for the approval of the PR
      </td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td><b>Yellows</b> are suggestions to improve the code, warnings and tips<br/>
      Not required for the approval of the PR, and should not be used for FTAR calculation.
      </td>
    </tr>
    <tr>
      <td>:white_check_mark:</td>
      <td><b>Green Findings</b> highlight good ideas and solutions</td>
    </tr>
    <tr>
      <td>:repeat:</td>
      <td><b>Repeated</b> instances of the same findings are counted just once and flagged with the repeat icon</td>
    </tr>
  </table>

  Feel free to reply to this comment, (make sure you `@tag` us) so we can start a conversation. The goal here is to learn and grow together and make our products better.

  > NOTE: This review **does not** replace your team's review._
</details>
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR increased your FTAR from 0.00 % to 100.00 %, good job, and for more quality improvements you can read about yellow findings.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>100.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>0</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

**Comment by vdefeo-xo** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

File: components/ide/code/leeway.Dockerfile
Comment (vdefeo-xo) [inline components/ide/code/leeway.Dockerfile:48]: <details>
<summary>:warning: <strong>Commented-out Code is Committed.</strong> (click for details)</summary>

This pull request contains commented code which is not recommended when a version control repository (e.g., GitHub) is used. It is recommended to delete all commented instructions before committing to VCS.

**[More details](https://confluence.devfactory.com/x/yAV7HQ)**

## The improvement suggestion in this case
Consider avoiding committing commented code.
</details>

[//]: # (<a id="f1caed9f-4626-459c-acf6-9411e0935d22" data-finding-id="494601672" href="none" codereview-uuid-reserved-tag data-business-id="d2a596db-d8bf-43dd-b073-6d1bacb7fbe5" />)

**Review by vdefeo-xo — COMMENTED** [review]
Hi, I am the IaC expert who performed the review of this Pull Request. We have well-defined [coding and review standards](https://confluence.devfactory.com/display/CodeReview/Findings), and you will find below suggestions to improve the code quality, based on these guidelines.
My additional comments are in the Notes section. I hope you find them useful.

<table>
  <tr>
    <td>:no_entry_sign:<b>Severes:</b>  0</td>
    <td>:exclamation:<b>Reds:</b>  0</td>
    <td>:warning:<b>Yellows:</b>  1</td>
    <td>:white_check_mark:<b>Greens:</b>  0</td>
  </tr>
</table>

<details><summary>More information</summary>
  <table>
    <tr>
      <td>:no_entry_sign:</td>
      <td><b>Severe Findings</b> point out serious violations which can affect stability, performance, maintainability and therefore should be fixed.
        <ul>
          <li>Please note that the merge might have been blocked until all Severe Findings are fixed.</li>
          <li>While it is highly recommended to implement all findings only Severe Findings might have been blocking the merge</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>:exclamation:</td>
      <td><b>Red Findings</b> represent potential bugs and it is highly recommended to implement them.<br/>
      Not required for the approval of the PR
      </td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td><b>Yellows</b> are suggestions to improve the code, warnings and tips<br/>
      Not required for the approval of the PR, and should not be used for FTAR calculation.
      </td>
    </tr>
    <tr>
      <td>:white_check_mark:</td>
      <td><b>Green Findings</b> highlight good ideas and solutions</td>
    </tr>
    <tr>
      <td>:repeat:</td>
      <td><b>Repeated</b> instances of the same findings are counted just once and flagged with the repeat icon</td>
    </tr>
  </table>

  Feel free to reply to this comment, (make sure you `@tag` us) so we can start a conversation. The goal here is to learn and grow together and make our products better.

  > NOTE: This review **does not** replace your team's review._
</details>
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR increased your FTAR from 0.00 % to 100.00 %, good job, and for more quality improvements you can read about yellow findings.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>100.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>0</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

File: components/ws-proxy/pkg/proxy/routes.go
Comment (sri-vathsa) [inline components/ws-proxy/pkg/proxy/routes.go:85]: Do not override existing icon files.

Comment (satyamkumar-cn) [reply components/ws-proxy/pkg/proxy/routes.go:85]: Here its not overriding the file. I am just mainting a path variable for the same . User can just switch the path as desired

Comment (pavan-tri) [reply components/ws-proxy/pkg/proxy/routes.go:85]: As less code changes as possible to main repo. We are not in code cleanup mode or future flexibility mode.

**Comment by alexander-a-chernyshev** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

Please consider using [Analyzer UI](https://confluence.devfactory.com/display/CodeReview/Analyzer+UI) extension to catch issues in pull requests.

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

**Comment by alexander-a-chernyshev** [comment]
:white_check_mark: No new issues have been introduced since the previous review.

**Good work!**
<details>
    <summary>T4W FTAR Statistics</summary>
    <br />
    CodeRead team introduced new quality measurement to evaluate the trailing 4 weeks First Time Acceptance Rate of Pull Requests, this PR increased your FTAR from 0.00 % to 100.00 %, good job, and for more quality improvements you can read about yellow findings.
	<br />
    -
    <details>
        <summary>More Details</summary>
        <br />
        <table>
            <tr>
                <td>Current FTAR</td>
                <td><b>100.00 %</b></td>
            </tr>
            <tr>
                <td>Total Reviewed Pull Requests for the last 4 weeks</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Passed</td>
                <td>1</td>
            </tr>
            <tr>
                <td>Rejected</td>
                <td>0</td>
            </tr>
        </table>
        More details about how FTAR is calculated, refer to this page - <a
            href="https://confluence.devfactory.com/x/Cw8dGg">FTAR - First time acceptance rate</a>
    </details>

</details>

**Pull request closed without merging**
